src/tasks/DockerTaskProvider.ts

```diff
@@ -47,7 +47,7 @@ export abstract class DockerTaskProvider implements TaskProvider {
                 context.actionContext = actionContext;
                 context.platform = getPlatform(task.definition);
 
-                context.actionContext.telemetry.properties.platform = context.platform;
+                context.actionContext.telemetry.properties.platform = context.platform.toString();

```

Comment:
We need to validate that this still works as expected when the extension is webpacked.

Reply:
I can see it with Fiddler. However, I see it even before this change so not sure what's happening...